lib/play-audio.coffee

```diff
@@ -2,7 +2,10 @@ path = require "path"
 
 module.exports =
   play: ->
-    pathtoaudio = path.join(__dirname, '../audioclips/gun.wav')
+    if (@getConfig "audioclip") is "customAudioclip"
+      pathtoaudio = path.join(__dirname,"../audioclips/" + @getConfig "customAudioclip")
```

Comment:
Here the custom path for the audio should not be relative to the package folder. To not force the user to find this folder as it's easier to copy the audio clip path and paste it in the setting.